test/aws/centos_repos.yml

```diff
@@ -0,0 +1,52 @@
+---
+# Note: OpenShift repositories under CentOS may be shipped through the
+# "centos-release-openshift-origin" package which configures the repository.
+# This task matches the file names provided by the package so that they are
+# not installed twice in different files and remains idempotent.
+- name: Ensure libselinux-python is installed
+  package:
+    name: libselinux-python
+    state: present
+  register: result
+  until: result is succeeded
+
+- name: Remove openshift_additional.repo file
+  file:
+    dest: /etc/yum.repos.d/openshift_additional.repo
+    state: absent
+
+- name: Create any additional repos that are defined
+  yum_repository:
+    description: "{{ item.description | default(item.name | default(item.id)) }}"
+    name: "{{ item.name | default(item.id) }}"
+    baseurl: "{{ item.baseurl }}"
+    gpgkey: "{{ item.gpgkey | default(omit)}}"
+    gpgcheck: "{{ item.gpgcheck | default(1) }}"
+    sslverify: "{{ item.sslverify | default(1) }}"
+    sslclientkey: "{{ item.sslclientkey | default(omit) }}"
+    sslclientcert: "{{ item.sslclientcert | default(omit) }}"
+    sslcacert: "{{ item.sslcacert | default(omit) }}"
+    file: "{{ item.name }}"
+    enabled: "{{ item.enabled | default('no')}}"
+  with_items: "{{ openshift_additional_repos }}"
```

Comment:
Are we using `openshift_additional_repos` in CI?  And if so, what repos are we adding?

Reply:
I think we are using the repo `origin-pr` here: https://github.com/openshift/release/blob/master/cluster/test-deploy/aws-4.0/vars.yaml